modules/code-deployment/src/main/java/org/apache/ignite/internal/deployunit/DeploymentUnitAccessorImpl.java

```diff
@@ -27,6 +30,8 @@
 public class DeploymentUnitAccessorImpl implements DeploymentUnitAccessor {
     private final RefCountedObjectPool<DeploymentUnit, DisposableDeploymentUnit> pool = new RefCountedObjectPool<>();
 
+    private final RefCountedObjectPool<DeploymentUnit, Lock> locks = new RefCountedObjectPool<>();
```

Comment:
Why do you use `RefCountedObjectPool` for locks? Do you need to count them?

Reply:
Yep, we have to remove unused locks